modules/oauth20/cms_oauth_20_workflow.e

```diff
@@ -20,7 +20,7 @@ feature {NONE} -- Initialization
 		do
 			initilize (a_consumer)
 			create config.make_default (a_consumer.api_key, a_consumer.api_secret)
-			config.set_callback (a_host + "/account/"+ a_consumer.callback_name)
+			config.set_callback (a_host + "/account/oauth-callback"+ a_consumer.callback_name)
```

Comment:
it depends on how we configure the callback_name, but we can add / after callback and the consumer callback name does no need to add /<name>.